[0.023s] Invoking command in '/home/shriram/Desktop/ros2Project/build/arduinobot_msgs': CMAKE_PREFIX_PATH=/opt/ros/humble /usr/bin/cmake --build /home/shriram/Desktop/ros2Project/build/arduinobot_msgs -- -j12 -l12
[0.046s] -- Found ament_cmake: 1.3.5 (/opt/ros/humble/share/ament_cmake/cmake)
[0.182s] -- Found std_msgs: 4.2.3 (/opt/ros/humble/share/std_msgs/cmake)
[0.202s] -- Found rosidl_generator_c: 3.1.5 (/opt/ros/humble/share/rosidl_generator_c/cmake)
[0.204s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.209s] -- Found rosidl_generator_cpp: 3.1.5 (/opt/ros/humble/share/rosidl_generator_cpp/cmake)
[0.217s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.229s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.267s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.401s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.401s]   execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name
[0.401s]   arduinobot_msgs --arguments-file
[0.401s]   /home/shriram/Desktop/ros2Project/build/arduinobot_msgs/rosidl_adapter__arguments__arduinobot_msgs.json
[0.401s]   --output-dir
[0.401s]   /home/shriram/Desktop/ros2Project/build/arduinobot_msgs/rosidl_adapter/arduinobot_msgs
[0.402s]   --output-file
[0.402s]   /home/shriram/Desktop/ros2Project/build/arduinobot_msgs/rosidl_adapter/arduinobot_msgs.idls)
[0.402s]   returned error code 1:
[0.402s] 
[0.402s]   Traceback (most recent call last):
[0.402s] 
[0.402s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.402s]       return _run_code(code, main_globals, None,
[0.402s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.402s]       exec(code, run_globals)
[0.402s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.402s]       sys.exit(main())
[0.403s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.403s]       abs_idl_file = convert_to_idl(
[0.403s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 24, in convert_to_idl
[0.403s]       return convert_srv_to_idl(
[0.403s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/srv/__init__.py", line 28, in convert_srv_to_idl
[0.403s]       srv = parse_service_string(package_name, input_file.stem, content)
[0.403s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 850, in parse_service_string
[0.403s]       raise InvalidServiceSpecification(
[0.403s] 
[0.403s]   rosidl_adapter.parser.InvalidServiceSpecification: Could not find separator
[0.403s]   '---' between request and response
[0.404s] 
[0.404s] Call Stack (most recent call first):
[0.404s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.404s]   CMakeLists.txt:13 (rosidl_generate_interfaces)
[0.404s] 
[0.404s] [0m
[0.404s] -- Configuring incomplete, errors occurred!
[0.404s] See also "/home/shriram/Desktop/ros2Project/build/arduinobot_msgs/CMakeFiles/CMakeOutput.log".
[0.407s] gmake: *** [Makefile:747: cmake_check_build_system] Error 1
[0.409s] Invoked command in '/home/shriram/Desktop/ros2Project/build/arduinobot_msgs' returned '2': CMAKE_PREFIX_PATH=/opt/ros/humble /usr/bin/cmake --build /home/shriram/Desktop/ros2Project/build/arduinobot_msgs -- -j12 -l12
